### PR TITLE
fix: change metamask flow for better clarity

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/electron-main/http.ts [hunk 1]
```diff
@@ -119,7 +119,8 @@ function handleRequests(req: IncomingMessage, res: ServerResponse, cb: Callback)
       return;
     }
   }
-  const basePath = checkIfDevelopment() ? path.join(__dirname, '..', 'public') : __dirname;
+  const dirname = path.dirname(new URL(import.meta.url).pathname);
```
Comment (kelsos) [inline frontend/app/src/electron-main/http.ts:122]: we are on node > 20.11 so we should be able to use `import.meta.dirname`

https://stackoverflow.com/a/50052194

File: frontend/app/src/components/accounts/blockchain/MetamaskAddressesImport.vue [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+const { isPackaged, metamaskImport } = useInterop();
+
+async function importAddresses() {
+  const addresses = await (isPackaged ? metamaskImport() : getMetamaskAddresses());
+  emit('update:addresses', addresses);
+}
+</script>
+
+<template>
+  <div>
```
Comment (kelsos) [inline frontend/app/src/components/accounts/blockchain/MetamaskAddressesImport.vue:33]: Is the extra div around absolutely needed? do we pass style here?

**Review by kelsos — COMMENTED** [review]
Ok so small comments plus we need to make sure that the address field doesn't keep the state when changing chains, especially if e.g. we go from ETH to BTC :)

**Comment by lukicenturi** [comment]
should be updated now

> Ok so small comments plus we need to make sure that the address field doesn't keep the state when changing chains, especially if e.g. we go from ETH to BTC :)

**Review by kelsos — APPROVED** [review]
lgtm

**Pull request merged**